packages/nextjs/components/ActivitiesItem.tsx

```diff
@@ -0,0 +1,225 @@
+        <div className="  w-1/6 flex-shrink-0">
+          {ensAvatar ? (
+            // Don't want to use nextJS Image here (and adding remote patterns for the URL)
+            // eslint-disable-next-line
+          <img
+            className="p-0.5"
+            src={ensAvatar}
+            width={65}
+            height={50}
+            alt={`${address} avatar`}
+          />
+          ) : (
+            <Blockies className="p-0.5" size={blockieSizeMap[size]} seed={address.toLowerCase()} scale={4} />
+          )}
+        </div>
+        <div className="w-4/6">
+          <div className="flex space-x-10 px-1 w-full">
+            <span className="h-8"> {address?.slice(0, 7)} </span>
+            <span className=""> Amount: {amount} ETH </span>
+          </div>
+          <Address address={address} />
+        </div>
+        <div className=" ml-6 w-1/6">
+          <span> {landedOn} </span>
+        </div>
+      </div>
+    </>
+  );
+};
+
+export type TWinnerItemProps = {
+  address: string;
+  amount: number;
+};
+export const WinnerItem = ({ address, amount }: TWinnerItemProps) => {
+  const [ensAvatar, setEnsAvatar] = useState<string | null>();
+
+  const [ens, setEns] = useState<string | null>();
+  const { data: fetchedEnsAvatar } = useEnsAvatar({
+    address,
+    enabled: isAddress(address ?? ""),
+    chainId: 1,
+    cacheTime: 30_000,
+  });
+
+  const blockieSizeMap = {
+    xs: 6,
+    sm: 7,
+    base: 8,
+    lg: 9,
+    xl: 10,
+    "2xl": 12,
+    "3xl": 15,
+  };
+
+  const size = "3xl";
+
+  useEffect(() => {
+    setEns(fetchedEnsAvatar);
+  }, [fetchedEnsAvatar]);
+
+  useEffect(() => {
+    setEnsAvatar(fetchedEnsAvatar);
+  }, [fetchedEnsAvatar]);
+
+  // Skeleton UI
+  if (!address) {
+    return (
+      <div className="animate-pulse flex space-x-4">
+        <div className="rounded-md bg-slate-300 h-6 w-6"></div>
+        <div className="flex items-center space-y-6">
+          <div className="h-2 w-28 bg-slate-300 rounded"></div>
+        </div>
+      </div>
+    );
+  }
+
+  if (!ethers.utils.isAddress(address)) {
+    return <span className="text-error">Wrong address</span>;
+  }
+
+  return (
+    <>
+      <div className="flex justify-between space-x-3 px-8 my-2 mb-5">
+        <div className="  w-1/6 flex-shrink-0">
+          {ensAvatar ? (
+            // Don't want to use nextJS Image here (and adding remote patterns for the URL)
+            // eslint-disable-next-line
+            <img
+              className="p-0.5"
+              src={ensAvatar}
+              width={65}
+              height={50}
+              alt={`${address} avatar`}
+            />
+          ) : (
+            <Blockies className="p-0.5" size={blockieSizeMap[size]} seed={address.toLowerCase()} scale={4} />
+          )}
+        </div>
+        <div className="w-3/6">
+          <div className="flex justify-between px-1 w-full">
+            <span className="h-8"> {address?.slice(0, 7)} </span>
```

Comment:
non needed

Reply:
Because of the Ui, I had to create a custom component to fit the UI  [Figma UI](https://www.figma.com/file/D0YZXzXhNtaP8j0lcq2JZR/Dice?type=design&node-id=1%3A2&mode=design&t=9nWbwZgT95vcsicq-1)